### PR TITLE
eldruin/bmi160-rs/issues/5 Add accel, gyro range and types

### PR DESCRIPTION
**Pull request by rland93**
Hi

I started a bit of work on #5 because I need to be able to set ranges explicitly. I have tested this on my own hardware but if there is any further testing work I am happy to do it, hardware willing.

### CODE REVIEW

**Review by eldruin — COMMENTED** [review]
Great work, thank you!
Could you add an entry to the changelog and note the new methods in the Readme/lib.rs overview?

**Comment by rland93** [comment]
I'm a big dummy and FORGOT about this PR.

will do soon then we can get it merged :)

**Comment by rland93** [comment]
@eldruin Done!

File: src/types.rs [hunk 1]
```diff
@@ -29,6 +29,19 @@ pub enum AccelerometerPowerMode {
     LowPower,
 }
 
+/// Accelerometer Range.
+/// See 2.11.12 Register (0x41) ACC_RANGE;
+/// BMI160 BST-BMI160-DS000-09 Rev 1.0 p. 59
```
Comment (eldruin) [inline src/types.rs:34]:
```suggestion
```
I think this may be unnecessarily detailed.

**Comment by avsaase** [comment]
@rland93 Do you plan to finish this PR? I want to implement the scaling as discussed in https://github.com/eldruin/bmi160-rs/issues/2 so the changes in this PR as useful to have.

**Pull request closed without merging**